### PR TITLE
S2U-7 Samigo add Assessment Report

### PR DESCRIPTION
**Pull request by JuanDavid102**

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 1]
```diff
@@ -301,8 +310,6 @@ public boolean histogramScores(HistogramScoresBean histogramScores, TotalScoresB
 					  submissionsSortedForDiscrim.get(numSubmissions-1-i)).getAgentId());
 		  }
 		  
-		  PublishedAssessmentIfc pub = (PublishedAssessmentIfc) pubService.getPublishedAssessment(publishedId, false);
-		  
 		  if (pub != null) {
```
[the remark this answers is not in the document]
Comment (JuanDavid102) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java]: Ok. I will remove it

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages.properties [hunk 2]
```diff
@@ -380,3 +380,15 @@ incorrect_answers_title=Incorrect Answers
 empty_answers_title=Empty Answers
 summary_title=Short Summary
 answers_title=Answers
+
+# S2U-7
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages.properties:384]:
```suggestion
```

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_ca.properties [hunk 3]
```diff
@@ -377,3 +377,15 @@ incorrect_answers_title=Respostes Incorrectes
 empty_answers_title=Respostes Buides
 summary_title=Resum breu
 answers_title=Respostes
+
+# S2U-7
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_ca.properties:381]:
```suggestion
```

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_es.properties [hunk 4]
```diff
@@ -377,3 +377,15 @@ incorrect_answers_title=Respuestas Incorrectas
 empty_answers_title=Respuestas Vac\u00edas
 summary_title=Resumen corto
 answers_title=Respuestas
+
+# S2U-7
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_es.properties:381]:
```suggestion
```

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties [hunk 5]
```diff
@@ -377,3 +377,15 @@ incorrect_answers_title=Erantzun Okerrak
 empty_answers_title=Erantzun Hutsak
 summary_title=Laburpen laburra
 answers_title=Erantzunak
+
+# S2U-7
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties:381]:
```suggestion
```

**Comment by jesusmmp** [comment]
It seems you need to change also these header licenses...

Warning: WARNING] Missing header in: /home/runner/work/sakai/sakai/samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/util/ExcelExportUtil.java
Warning: WARNING] Missing header in: /home/runner/work/sakai/sakai/samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/util/PdfExportUtil.java
Warning: WARNING] Missing header in: /home/runner/work/sakai/sakai/samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/model/AssessmentReport.java
Warning: WARNING] Missing header in: /home/runner/work/sakai/sakai/samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/model/AssessmentReportSection.java
Warning: WARNING] Missing header in: /home/runner/work/sakai/sakai/samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/servlet/evaluation/ExportReportServlet.java

**Comment by JuanDavid102** [comment]
Ok. Thanks i will do now all the modifications

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thank you very much @JuanDavid102 !

**Pull request merged**